hudi-kafka-connect/pom.xml

```diff
@@ -271,5 +271,12 @@
             <artifactId>junit-platform-commons</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
```

Comment:
I see this dependency is required to set env vars at test time. Is there a way to avoid this? Maybe add a wrapper class for system.getEnv calls and mock that class in test?

Reply:
Besides, this dependency uses MIT License.  Not sure if we allow it.

Reply:
@codope @yihua 
I reconsidered and modified the codes. Use `System.env` to get environment variables should not be necessary in these test cases, so I deleted dependencies and related codes.